metricbeat/module/kubernetes/util/kubernetes.go

```diff
@@ -157,6 +158,10 @@ func NewResourceMetadataEnricher(
 		},
 	)
 
+	if _, ok := res.(*kubernetes.Pod); ok {
+		enricher.isPod = true
+	}
```

Comment:
Could you add a comment here or where the `isPod` variable is used explaining a little bit why this is needed?

Reply:
pushed e17c704